.github/workflows/e2e-spin-deploy.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    if: ${{ github.repository_owner == 'fermyon' }}
```

Comment:
Are the changes here intended to be part of this PR as well?

Reply:
yeah. The secrets are not available to forks, thus Github action runs (that need secrets) would fail on PR from fork.